lecture_2/homework/app/models/clan.rb

```diff
@@ -1,4 +1,4 @@
 class Clan < ApplicationRecord
-  has_many :samurais, dependent: :destroy
+  has_many :warriors, dependent: :destroy
   validates :name, presence: true
```

Comment:
W migracji dodałeś unikalny indeks na nazwę klanu. Żeby aplikacja nie rzucała 500 tylko sensownie to obsługiwała musimy jeszcze dodać odpowiednią walidację na modelu